ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/StreamsTaskMetadata.java

```diff
@@ -59,7 +59,7 @@ public static StreamsTaskMetadata fromStreamsTaskMetadata(final TaskMetadata tas
       );
     }
     return new StreamsTaskMetadata(
-        taskMetadata.taskId(),
+        taskMetadata.taskId().toString(),
```

Comment:
Should we really pass in a `String` or update `StreamsTaskMetadata` to accept a `TaskId`?

Reply:
Yes this is just a wrapper class and should probably reflect the TaskMetadata. Though I am worried about what changing it will to the the ksqlHC

Reply:
I don't know enough about how this class is used to say how changing it from `String` to `TaskId` would do. On a similar note, I noticed (or rather, IntelliJ. noticed) that a number of the methods on this class are completely unused in the codebase. I guess that makes sense if they're public APIs used by the healthcheck, and I don't really know how (or if) the healthcheck does testing, but maybe we should have some test coverage within ksql too?

Reply:
The HC I don't think test this directly but it would cause some tests to fail so we would not worry about the HC not working unexpectedly. I think it might be better to just file a ticket to update the StreamsTaskMetadata and its uses in the HC and leave it as a String for now